src/generated/stellar-xdr_generated.js

```diff
@@ -1861,7 +1861,7 @@ xdr.struct("DontHave", [
 //   case SCP_QUORUMSET:
 //       SCPQuorumSet qSet;
 //   case SCP_MESSAGE:
-//       StellarMessage envelope;
+//       SCPMessage envelope;
```

Comment:
Should be `SCPEnvelope`. Please do not edit autogenerated files manually. This file was generated using [xdrgen](https://github.com/stellar/xdrgen).

Reply:
Thank you for the info. I commited a new run of `rake xdr:update`. This also fixed the problem in xdr/stellar-overlay